### ISSUE TITLE
KeyError: u'title'

### ISSUE THREAD

**Issue by stevezau** [croneter/PlexKodiConnect#982]
## Describe the bug
Just starting getting KeyError: u'title'

I've a complete reset from scratch. Also updated to 2.9.4 beta, same error.

I did notice the plugin was defusedxml plugin was recently updated.. not sure if it's related.
```
2019-09-07 12:39:20.239 T:139753971234560   ERROR: PLEX.widget: xml that caused the crash: "Video": {u'grandparentTitle': u'Kings Cross ER', u'librarySectionID': u'2', u'art': u'/library/metadata/6748/art/1567058875', u'key': u'/library/metadata/6750', u'grandparentKey': u'/library/metadata/6748', u'parentTitle': u'Season 3', u'addedAt': u'1444222800', u'year': u'2015', u'ratingKey': u'6750', u'parentIndex': u'3', u'librarySectionKey': u'/library/sections/2', u'updatedAt': u'1567058875', u'duration': u'2699800', u'guid': u'com.plexapp.agents.thetvdb://255788/3/5?lang=en', u'grandparentRatingKey': u'6748', u'index': u'5', u'grandparentThumb': u'/library/metadata/6748/thumb/1567058875', u'thumb': u'/library/metadata/6750/thumb/1567058875', u'title': u'Episode 5', u'parentKey': u'/library/metadata/6749', u'parentRatingKey': u'6749', u'contentRating': u'TV-MA', u'originallyAvailableAt': u'2015-10-08', u'parentGuid': u'com.plexapp.agents.thetvdb://255788/3?lang=en', u'grandparentGuid': u'com.plexapp.agents.thetvdb://255788?lang=en', u'librarySectionTitle': u'TV Shows', u'type': u'episode', u'summary': u'', u'grandparentArt': u'/library/metadata/6748/art/1567058875'}
2019-09-07 12:39:20.239 T:139753971234560   ERROR: PLEX.utils: Error encountered:  - u'title'
2019-09-07 12:39:20.239 T:139753971234560   ERROR: PLEX.utils: _____________________________________________________________
2019-09-07 12:39:20.240 T:139753971234560   ERROR: PLEX.utils:     Traceback (most recent call last):
2019-09-07 12:39:20.240 T:139753971234560   ERROR: PLEX.utils:       File "/home/media/.kodi/addons/plugin.video.plexkodiconnect/resources/lib/widgets.py", line 72, in generate_item
2019-09-07 12:39:20.240 T:139753971234560   ERROR: PLEX.utils:         return _generate_content(api)
2019-09-07 12:39:20.240 T:139753971234560   ERROR: PLEX.utils:       File "/home/media/.kodi/addons/plugin.video.plexkodiconnect/resources/lib/widgets.py", line 211, in _generate_content
2019-09-07 12:39:20.240 T:139753971234560   ERROR: PLEX.utils:         item['label'] = item['title']
2019-09-07 12:39:20.240 T:139753971234560   ERROR: PLEX.utils:     KeyError: u'title'
2019-09-07 12:39:20.240 T:139753971234560   ERROR: PLEX.utils: _____________________________________________________________
2019-09-07 12:39:20.534 T:139754006103808  NOTICE: PLEX.default: plugin.video.plexkodiconnect stopped
```

## To Reproduce
1. Try play and video or go into any part of PlexKodiConnect browsing the plex files

## Expected behavior
It to work :)

**Comment by croneter** [comment]
Please share your entire DEBUG log as explained in the Github wiki. I cannot reproduce your KeyError, for some reason...

**Comment by BrutuZ** [comment]
Started getting a couple of these recently. I suspect it being related to "illegal" characters in the title since widget entries with those, namely "Fate\Stay Night", "Fate\Grand Order" and "Re:Zero", are being rendered as posters only, without the text element.

**Comment by croneter** [comment]
@BrutuZ Log? Could you also provide links to the IMDB.com movie entry that you suspect?

**Comment by BrutuZ** [comment]
After digging a little deeper, turns out my initial guess was wrong. Did a full database reset\rebuild via PKC and the error stopped manifesting. The widgets for the aforementioned items still don't show the text labels, but that's apparently another issue.
I did manage to reproduce the KeyError issue though. All it took was for me to refresh an item's metadata inside Kodi. The reason I did that before was the [constant failing of Plex's metadata refresh from TheTVDB](https://forums.plex.tv/t/cant-refresh-metadata/456635), and since I also keep externally generated _.nfo_ files along the media (which will have the correct\updated info), Kodi prioritizes those when a refresh is called from the interface, which in turn breaks PKC parsing.
For reference: I do use direct paths instead of addon playback mode on PKC
